knack/cli.py

```diff
@@ -104,15 +104,15 @@ def __init__(self,
     def _should_show_version(args):
         return args and (args[0] == '--version' or args[0] == '-v')
 
-    def get_cli_version(self):  # pylint: disable=no-self-use
+    def get_cli_version(self):
```

Comment:
The removal of `# pylint: disable=no-self-use` is explained in https://github.com/Azure/azure-cli-dev-tools/pull/359#discussion_r1259334166